### PR TITLE
build: publish list of images for addons

### PR DESCRIPTION
**Pull request by dkoshkin**
Added automation to publish a yaml with a list of images needed for each of the addons.

Suggestions on the format of the file are welcome!

### CODE REVIEW

File: addon-images.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+aws-cpi-v1.27:
```
Comment (supershal) [inline addon-images.yaml:3]: How are we plan to consume this file?

Comment (dkoshkin) [reply addon-images.yaml:3]: Great question! The idea is that any consumer (e.g. konvoy2) can run a simple `yq` query to get the list of images.

```
 yq '.*.images' addon-images.yaml
 ```
 
 or 
 
 ```
 yq '.*.images' addon-images.yaml | tr -d ' -'
 ```

Comment (supershal) [reply addon-images.yaml:3]: nice. each client wont have to build tooling to scrape images for bundling.

**Comment by supershal** [comment]
The image file format in the PR is well readable. Should we consider the image file format used by mindthegap? https://github.com/mesosphere/mindthegap/blob/main/images-example.yaml 
I am just thinking it might be easier to consume it by the `dkp` CLI directly. The only thing is we will lose some details with mindthegap format.

**Comment by dkoshkin** [comment]
Will discuss offline and reopen later

**Pull request closed without merging**